src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp

```diff
@@ -93,9 +93,13 @@ bool PerformanceImplicitGemm::IsValid(const ConvolutionContext& ctx) const
     }
 
     // divide block work by [K, B]
-    if(!(K % KPerBlock == 0 && B % BPerBlock == 0 && E % (2 * EPerBlock) == 0))
+    if(!(K % KPerBlock == 0 && B % BPerBlock == 0 && E % EPerBlock == 0))
         return false; // wrong! cannot divice N evenly among thread
 
+    // WORKAROUND_ISSUE_659
+    if(E % (2 * EPerBlock) != 0)
+        return false;
```

Comment:
Could you please
```suggestion
#if WORKAROUND_ISSUE_659
    if(E % (2 * EPerBlock) != 0)
        return false;
#endif
```
and
```C++
#define WORKAROUND_ISSUE_659 1
```
at the beginning of this file?

Reply:
Pushed. Sorry for forgetting this.

Reply:
NP!